kura/org.eclipse.kura.net.configuration/src/main/java/org/eclipse/kura/net/configuration/NetworkConfigurationServiceCommon.java

```diff
@@ -457,6 +458,13 @@ private static void addMtuDefinition(Tocd tocd, String ifaceName) {
         tad.setRequired(true);
         tocd.addAD(tad);
     }
+    
+    private static void addPromiscDefinition(Tocd tocd, String ifaceName) {
+        Tad tad = buildAttributeDefinition(String.format(PREFIX + "%s.config.promisc", ifaceName),
+                NetworkConfigurationPropertyNames.CONFIG_PROMISC, Tscalar.INTEGER);
+        tad.setRequired(true);
```

Comment:
What happens if it is not provided? There's no default provided either so I expect bad things to happen in the old networking 🤔

Reply:
This could probably be a `.setRequired(false)`. The idea is the Kura should do nothing if the promisc parameter is not populated. And it's actually working fine with required fields being missing 🤔

Reply:
🤔  @pierantoniomerlino thoughts?

Reply:
IMO, it is sufficient to add a default value (probably -1 is the right choice).
```
tad.setDefault("-1");
```